CMakeLists.txt

```diff
@@ -365,6 +365,14 @@ if(BUILD_TESTS)
       CURL_CLIENT TRUE
     )
 
+    add_e2e_test(
+      NAME js_end_to_end_logging
+      PYTHON_SCRIPT ${CMAKE_SOURCE_DIR}/tests/e2e_logging.py
+      ADDITIONAL_ARGS
+        --js-app-script ${CMAKE_SOURCE_DIR}/src/apps/logging/loggingjs.lua
+      CURL_CLIENT TRUE
```

Comment:
Minor: I don't think that the curl client is particularly useful here?

Reply:
Oh no, I agree, it's just because it's there in the other loggingenc one, but I can remove it.